File: php/WesleySafadao.php
<?php

function wesley_safadao($day, $month, $year) {
  $safadeza = array_sum(range(1, $month)) + ($year / 100) * (50 - $day);
  $anjo = 100 - $safadeza;
  echo "Você é {$safadeza}% safado e {$anjo}% anjo";
}

File: clojure/wesley_safadao.clj
(defn sum_month [n]
  (->> (range (+ n 1))
    (reduce +)))

(defn safadeza [day month year]
  (+ (sum_month month)
     (* (/ year 100)
        (- 50 day))))

(defn anjo [saf]
  (- 100 saf))

(defn wesley_safadao [day month year]
  (let [saf (safadeza day month year)
        angel (anjo saf)]
    (println (str "Você é " (float saf) "% safado e " (float angel) "% anjo"))))
